ui/app/AppLayouts/Chat/panels/UserListPanel.qml

```diff
@@ -88,7 +88,7 @@ Item {
                 asset.isLetterIdenticon: (asset.name === "")
                 asset.color: Utils.colorForColorId(model.colorId)
                 status: model.onlineStatus
-                ringSettings.ringSpecModel: Utils.getColorHashAsJson(model.pubKey, ensVerified) // FIXME: use model.colorHash
+                ringSettings.ringSpecModel: JSON.parse(model.colorHash)
```

Comment:
In this case the model or backend should take care of it IMO. Will check how the ens verified contact rings are handled in nim.

Reply:
It makes sense to let the backend handle it like Alex did. 

However, `JSON.parse` is a finicky function and throws when it doesn't like what is given to it, so we need a `try/catch` somewhere to make sure we handle those cases.
If you want, you can modify `StatusMemberListItem` to accept only a string and it does the parsing itself, so we don't have to do the add the parse everywhere we use `StatusMemberListItem`.

Another possibility for a faster short term solution is to just modify `getColorHashAsJson` to let it accept the `colorHash` so we don't have to call the backend for the info.

Reply:
> If you want, you can modify `StatusMemberListItem` to accept only a string and it does the parsing itself, so we don't have to do the add the parse everywhere we use `StatusMemberListItem`.
> 
> Another possibility for a faster short term solution is to just modify `getColorHashAsJson` to let it accept the `colorHash` so we don't have to call the backend for the info.

Good point, thanks! I'll look into it and update. At this point I tend to think that StatusIdenticonRingSettings could receive the string color hash and do the parsing.

Reply:
@caybro @jrainville I've updated the colorHash parsing and handling in case on ENS verified contacts, please have a look.
The parsing itself is done in StatusIdenticonRingSettings.qml and the backend will not provide a colorHash if the contact is ENS verified.